src/LogicApp/LogicApp/help/Get-AzIntegrationAccountAssembly.md

```diff
@@ -0,0 +1,159 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.LogicApp.dll-Help.xml
+Module Name: Az.LogicApp
+online version: https://docs.microsoft.com/en-us/powershell/module/az.logicapp/get-azintegrationaccountassembly
+schema: 2.0.0
+---
+
+# Get-AzIntegrationAccountAssembly
+
+## SYNOPSIS
+Gets an integration account assembly.
+
+## SYNTAX
+
+### ByIntegrationAccount (Default)
+```
+Get-AzIntegrationAccountAssembly -ResourceGroupName <String> -ParentName <String> [-Name <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByInputObject
+```
+Get-AzIntegrationAccountAssembly [-Name <String>] -ParentObject <IntegrationAccount>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### ByResourceId
+```
+Get-AzIntegrationAccountAssembly [-Name <String>] -ParentResourceId <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Get-AzIntegrationAccountAssembly** cmdlet gets an assembly from an integration account.
+
+## EXAMPLES
+
+### Example 1: Get an assembly by parameters
+```powershell
+PS C:\> Get-AzIntegrationAccountAssembly -ResourceGroupName "sampleResourceGroup" -IntegrationAccountName "sampleIntegrationAccount" -AssemblyName "sampleAssembly"
```

Comment:
Please add example output for all of the examples (except the two remove cmdlets as they have no output)

Reply:
Added. They all look kind of generic though...

Reply:
That's totally fine